Yoast/wordpress-seo
# Fix the term edit metabox errors

## Summary

This PR can be summarized in the following changelog entry:

* Removed errors from the Term pages.

## Relevant technical choices:

* Removed the cornerstone toggle from the sidebar in term-edit pages.
* Removed the upsell from term-edit pages.
* Made sure the content analysis updates on keyword update.
* Added the dispatches for overall scores to the postDataCollector.

## Test instructions

This PR can be tested by following these steps:

* checkout this branch.
* Set up this branch:
   - `yarn install`
   - `Grunt build:js && grunt build:css`
   - `yarn start`
* Visit a term-edit page ( the page of a single category, for instance ).
* Make sure you have the feature flag active: `define( 'YOAST_FEATURE_GUTENBERG_SIDEBAR', true );
`
* Check if the metabox doesn't crash on the term-edit page.
* Check that all the following applies ( 🍎):
   - There's no "add another keyword" collapsible, like on a post-edit page.
   - There's no "cornerstone content" collapsible, like on the post-edit page.
   - When you update the keyword in the metabox, the analysis changes to reflect the new keyword.
   - The icons in the collapsibles of readabiliy and focus keyword analysis are present.

## Quality assurance


Fixes #10469
